sdk/messaging/azservicebus/admin/admin_client_queue.go

```diff
@@ -161,32 +142,21 @@ type UpdateQueueOptions struct {
 }
 
 // UpdateQueue updates an existing queue.
-func (ac *Client) UpdateQueue(ctx context.Context, queueName string, properties QueueProperties, options *UpdateQueueOptions) (*UpdateQueueResponse, error) {
-	newProps, resp, err := ac.createOrUpdateQueueImpl(ctx, queueName, &properties, false)
+func (ac *Client) UpdateQueue(ctx context.Context, queueName string, properties QueueProperties, options *UpdateQueueOptions) (UpdateQueueResponse, error) {
```

Comment:
I ran into this issue with KV. If a customer uses `CreateQueue`, they cannot use the embedded `QueueProperties` as an input in this `UpdateQueue` method. Sending an e-mail to discuss not embedding some of these models.

Reply:
Why can't they use QueueProperties as input?

Reply:
(it's an embedded property of GetQueueResponse)

```go
type GetQueueResponse struct {
	QueueProperties
}
```

Reply:
Ignore my comment, you're in the clear here

Reply:
Framing this for use in the future.